erpnext/crm/doctype/appointment/appointment.py

```diff
@@ -240,3 +242,14 @@ def _get_employee_from_user(user):
 	if employee_docname:
 		return frappe.get_doc("Employee", employee_docname)
 	return None
+
+
+def add_agent_assignment(args):
```

Comment:
```suggestion
def assign_agents(doctype: str, name: str, agents: list[str]) -> None:
```

Maybe this signature would improve readability a bit.